pyomo/contrib/alternative_solutions/lp_enum.py

```diff
@@ -77,15 +74,20 @@ def enumerate_linear_solutions(
         Boolean indicating that the solver output should be displayed.
     seed : int
         Optional integer seed for the numpy random number generator
+    pool_manager : None
+        Optional pool manager that will be used to collect solutions
 
     Returns
     -------
-    solutions
-        A list of Solution objects.
-        [Solution]
+    pool_manager
+        A PyomoPoolManager object
     """
     logger.info("STARTING LP ENUMERATION ANALYSIS")
 
+    assert num_solutions >= 1, "num_solutions must be positive integer"
```

Comment:
Argument checking of public APIs should raise exceptions and not use assertions (running python in optimized mode will skip *all* assertions).